README.md

```diff
@@ -345,6 +345,16 @@ be sure to clear your cookies.
 
 **NOTE**: At this time, it's not possible to sign in using this approach.
 
+### Working with UX Mocks
+
+When implementing user interfaces you will need to refer to the
```

Comment:
Minor note: I don't think we need to wrap README prose because GiHub will give us nicer text diffs if we don't and also it's kind of a pain. But do whatever you prefer.